homework/week1/2025320048-yujuyeon/[Homework][Week1] 1027번.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+typedef struct {
+    int h;
+    int num;
+} bldg;
+
+int left(bldg b[], int idx) {
```

Comment:
변수명을 보다 자세히 적으면 좋을 것 같습니다.
```suggestion
int cal_visible_buildings(bldg buildings[], int idx) {
```